src/features/offer/components/VenueSelectionList/VenueSelectionList.stories.tsx

```diff
@@ -12,7 +12,12 @@ const DynamicTemplate: ComponentStory<typeof VenueSelectionList> = (props) => {
   const [selectedItem, setSelectedItem] = useState<number>()
 
   return (
-    <VenueSelectionList {...props} selectedItem={selectedItem} onItemSelect={setSelectedItem} />
+    <VenueSelectionList
+      {...props}
+      selectedItem={selectedItem}
+      onItemSelect={setSelectedItem}
+      onRefresh={undefined}
```

Comment:
Ce serait pas mal d'avoir un 
```suggestion
      onRefresh={action('refresh')}
```

Reply:
Pourquoi pas mais quelle est l'intérêt dans la story ?